src/Monolog/Formatter/LogzIoFormatter.php

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Monolog\Formatter;
+
+/**
+ * Encodes message information into JSON in a format compatible with Logz.io.
+ *
+ * @author Christian Brückner <chris@chrico.info>
+ */
+class LogzIoFormatter extends JsonFormatter
+{
+
+    /**
+     * yyyy-MM-dd'T'HH:mm:ss.SSSZ
+     */
+    const DATETIME_FORMAT = "c";
```

Comment:
If logz.io use `format('c') `, you should use that I guess. I was a bit surprised that they don't support/use microseconds :)